### PR TITLE
Update ways_of_working.md - shifting membership

### PR DESCRIPTION
**Pull request by aleesteele**
### Summary

Shifting a few names and titles around on the Ways of Working document - am I missing anything?

@BatoolMM and @acocac - is everything up to date with the translation & localisation team, and Alejandro, not sure how your title should be accounted for now :)

### CODE REVIEW

**Comment by aleesteele** [comment]
Small flag – if either of you could approve this PR, that'd be great @BatoolMM and @acocac 😄 Thanks to you both!

File: ways_of_working.md [hunk 1]
```diff
@@ -29,7 +28,6 @@ It further outlines expectations and responsibilities with regard to working on
 | Sarah Gibson <br>([@sgibson91](https://github.com/sgibson91)) | Open Source Infrastructure Engineer, 2i2c & JupyterHub Community Development | in-kind contribution, The Turing Way Infrastructure Maintainers Co-lead | 2019 - Present | Prevously: Research Data Scientist, Nov 2018 - Aug 2021 (40%) |
 | Callum Mole <br>([callummole](https://github.com/callummole)) | Research Data Scientist, The Alan Turing Institute | Turing Binder Federation Membership - Project Contact | 2021 - Present | - |
 | Luke Hare <br>([lukehare](https://github.com/lukehare)) | Research Data Scientist, The Alan Turing Institute | Turing Binder Administrator | 2021 - Present | - |
-| Camila Rangel-Smith <br>([crangelsmith](https://github.com/crangelsmith)) | Senior Research Data Scientist, The Alan Turing Institute | The Turing Way Translation and Localisation Co-Lead | 2020 - Present | - |
 |  Batool Almarzouq <br>([BatoolMM](https://github.com/BatoolMM)) | Postdoctoral Researcher, KAIMRC and Honorary Research Fellow, The University of Liverpool  | The Turing Way Translation and Localisation Co-Lead | 2020 - Present | - |
```
Comment (BatoolMM) [inline ways_of_working.md:31]:
```suggestion
|  Batool Almarzouq <br>([BatoolMM](https://github.com/BatoolMM)) | The Alan Turing Institute, The University of Liverpool  | The Turing Way Translation and Localisation Co-Lead | 2020 - Present | - |
```

**Review by BatoolMM — APPROVED** [review]

File: ways_of_working.md [hunk 2]
```diff
@@ -29,7 +28,6 @@ It further outlines expectations and responsibilities with regard to working on
...
 | Alejandro Coca Castro <br>([acocac](https://github.com/acocac)) | Postdoctoral Research Associate, The Alan Turing Institute | The Turing Way Translation and Localisation Co-Lead | 2021 - Present | - |
```
Comment (acocac) [inline ways_of_working.md:32]:
```suggestion
| Alejandro Coca Castro <br>([acocac](https://github.com/acocac)) | Research Fellow, The Alan Turing Institute | Environment & Sustainability Co-Lead | 2023 - Present | Previously: The Turing Way Translation and Localisation Co-Lead, 2021 - 2023 (in-kind contribution) |
```

**Review by acocac — APPROVED** [review]

**Pull request merged**